script_vts.py

```diff
@@ -18,184 +18,160 @@ def exception_handler(print_exception=False, exception=""):
         del exc_type, exc_obj
         fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
         print("Exception on line: ", exc_tb.tb_lineno, " in ", fname)
-    return
+    return None
 
 
-def checking_arguments(arguments):
+def verify_at_least_one_command_line_argument(arguments):
     """Function checking the number of arguments"""
     if len(arguments) > 1:
         return arguments
-    return
+    return None
+
+
+def parse_arguments():
+    """Function used to parse arguments"""
+    parser = argparse.ArgumentParser()  # Create the parser to have arguments in the script
+    parser.add_argument('--string', type=str, )  # Adding string argument
+    parser.add_argument('--file', type=argparse.FileType('r'))  # Adding File argument
+    return parser.parse_args()  # Parse the argument
 
 
 def get_path(prompt):
     """Get Path with input(), check if both path/file exist"""
-
     try:
         prompt = input("Please input the absolute path of your API key configuration file: ")
-        if os.path.exists(prompt):
-            print("[+] The path is valid!")
-            prompt = Path(prompt)
-
-            if prompt.is_file():
-                print("[+] We did find the file!")
-            else:
-                print("[-] We didn't found the file!")
-                return
-        else:
-            print("[-] The path is not valid!")
-            return
+        prompt = checking_path(prompt)
     except ValueError as val_err:
         exception_handler(True, val_err)
-        return
-
+        return None
     return prompt
 
 
+def checking_path(raw_path):
```

Comment:
Great function. Let's rename it to something more generic such as: `validate_path` or simply `check_if_filepath_exists`.

Additionally let's try to strip out the references to the API file specifically. That way you can use this function for any future programs.